apps/bfd-server/bfd-server-war/src/test/resources/openapi/posts.yaml

```diff
@@ -354,3 +354,241 @@ paths:
+                identifier:
+                  type: string
+                  includeTaxNumbers: |-
+                    Setting this flag to _true_, provides tax number in the EOB transformed data for the response.
+                    
+                    Example:
+                      - `includeTaxNumbers=true`
+                  example: true
+                isHashed:
+                  type: string
+                  description: |-
+                    Setting this flag to _true_, provides tax number in the EOB transformed data for the response.
+                    
+                    Example:
+                      - `isHashed=true`
+                            example: "ABCD"
+                service-date:
+                  type: string
+                  description: |-
+                    Only satisfy the Search request if a claim's Date
+                    falls within a specified _DateRange_. A _DateRange_ can be
+                    defined by providing less than `lt` and/or greater than `gt` values.
+                    This parameter can be included in a request one or more times.
+                    
+                    Examples:
+                      - `service-date=gt2023-01-02&service-date=lt2023-05-01`
+                      - `service-date=gt2023-01-02`
+                      - `service-date=lt2023-05-01`
+                  example: "lt2023-05-01"
+                startIndex:
+                  type: string
+                  description: |-
+                    When fetching a _Bundle Response_ using pagination, this URL parameter represents an offset
+                    (starting point) into the list of elements for the _Request_.
+                    It is optional and defaults to 1 if not supplied.
+                    A value 0 is not allowed and negative indices are not currently supported.
+                    
+                    Example:
+                    - `startIndex=100`
+                  example: 100
+                type:
+                  type: string
+                  description: |-
+                    A list of one or more comma-separated claim types to be included in the request;
+                    within BFD, the claim types represent an _OR_ inclusion logic meaning any claims matching one of the specified
+                    claim types will be checked
+                    
+                    Supported Claim Type values:
+                      - `fiss`
+                      - `mcs`
+                    
+                    Examples:
+                      - `type=fiss,mcs`
+                      - `type=fiss`
+                  example: "fiss,mcs"
+        required: true
+      responses:
+        '200':
+          description: Success
+          content:
+            application/fhir+json:
+              schema: { $ref: '#/components/schemas/FHIR-JSON-RESOURCE' }
+            application/fhir+xml:
+              schema: { $ref: '#/components/schemas/FHIR-XML-RESOURCE' }
```

Comment:
There's _a very_ minor issue with the no newline at EOF warning here. Could you make adjustments to your editor, write/save, and add a new commit to address this?

https://thoughtbot.com/blog/no-newline-at-end-of-file#following-the-rules-in-your-editor